src/playground/sample-data-ingestion/generator.py

```diff
@@ -0,0 +1,187 @@
+from faker import Faker
+
+# Standard libraries
+import shutil
+import json
+import gzip
+import csv
+import ast
+import os
+
+# Function that generates fake values for given fields
+# This is based off of the Faker library 
+def chooseField(kind, args=None):
```

Comment:
Would you like to define a class ? it is better for code organization and reusable.

Reply:
I'm not certain if that is necessary. I looked at another ingestion script and they did not use classes in their code. Additionally, this is a small component of my tool (there are only 2 functions).

Reply:
Yes, I designed my tool with 3 aspects or "subtools" in mind:
- A data generator utility
- A data ingestion tool (which may or may not use generated data) and connects to the OS client
- A plugin resource initializer which also connects to the OS client

This draft PR is for the data generator utility. I've talked to various plugin teams and from the requirements they gathered, I'm sure that the data generator utility is adequate. The ingestion and initializer will require some more feedback and modifying, of which I have scheduled a couple of 1:1s with other teams.

Reply:
Ok. I recommend you to read some articles about the "[Advantages of Using Classes in Python](https://intellipaat.com/blog/tutorial/python-tutorial/python-classes-and-objects/#no3)". I'm fine with function without class if you only have two function here.

Reply:
Will read, I haven't mentioned it yet but I will be defining classes for my plugin resource initializer and ingestion tools.